Gemfile.lock

```diff
@@ -20,3 +20,9 @@ PLATFORMS
 
 DEPENDENCIES
   rspec (~> 3.0.0.beta2)
+
+RUBY VERSION
+   ruby 2.0.0p648
```

Comment:
:-) all of our stuff is on ruby 2.2, so you'll save yourself some trouble later by switching over now

Reply:
The Gemfile specified 2.0 so for some reason I switched ruby versions as opposed to updating the Gemfile. In hindsight, that wasn't my most efficient option